### DISCUSSION TITLE
Potential differences in performance when using CI tools

### DISCUSSION THREAD

**Discussion by dkarzon** [GoogleChrome/lighthouse-ci#670] [General]
Hey, I've been running LHCI on Bitbucket Pipelines for a while now and I have been seeing differences in performance values that are outside my control (Bitbucket Infrastructure not my own) and I was wondering if there is a way to ensure more consistent performance across different hardware?

My bitbucket pipelines are set to use the same image and same versions of the tools connecting to the same server but I am seeing on the reports the CPU/Memory Power various a bit (~900 up to ~1300). This doesn't skew the results too much but would like to make sure it's as consistent as possible.
I am currently using the default settings for throttling when running collect.

**Comment by patrickhulce** [comment]
Have you already read through the [variability documentation in Lighthouse](https://github.com/GoogleChrome/lighthouse/blob/7b4816a1ca6cfa98f89707582755fcaf52b0fa5e/docs/variability.md#strategies-for-dealing-with-variance)? There are some useful strategies there though LHCI is already implementing some of those by default.

If you've already exhausted all of those options, my other advice would be to attempt to detect the strength of the machine *before* Lighthouse CI is run so you can adjust the throttling multiplier accordingly ([the benchmarkIndex code](https://github.com/GoogleChrome/lighthouse/blob/7b4816a1ca6cfa98f89707582755fcaf52b0fa5e/lighthouse-core/lib/page-functions.js#L198-L252)).
